src/ui/CustomMenuUI.js

```diff
@@ -295,15 +295,17 @@ class CustomMenuUI extends React.PureComponent<any, any> {
               delete val.runtime;
               if (1 === mode) {
                 // update
-                customStyles = runtime.saveStyle(val);
+                delete val.editorView;
+                customStyles = runtime.saveStyle(val).then((result) => {
+                  console.log(result);
```

Comment:
Don't spam logs with unnecessary success messages.  Here you log the whole list of styles.